sdks/java/core/src/test/java/org/apache/beam/sdk/io/TextIOTest.java

```diff
@@ -432,7 +436,7 @@ public void testPrimitiveReadDisplayData() {
   }
 
   private static Predicate<List<String>> haveProperHeaderAndFooter(final String header,
-                                                                   final String footer) {
+      final String footer) {
```

Comment:
No need to revert this code, because the changes are fairly minor. But:

In general, we prefer pull requests to only change formatting on the lines they change. This behavior increases readability of the PR itself and also helps in triage later when we're tracking back which PRs changed specific code.

The alternative is formatting-only (no functional code changes) PRs that touch large swaths of the code base.